website/src/pages/docs/features/subscriptions.mdx

```diff
@@ -367,7 +380,33 @@ const network = Network.create(executeQueryOrMutation, executeSubscription)
 
 ## GraphQL over Server-Sent Events Protocol (via `graphql-sse`)
 
-In case you want the subscriptions to be transported following the [GraphQL over Server-Sent Events Protocol](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md), you simply use the `@graphql-yoga/plugin-graphql-sse` plugin for GraphQL Yoga that exposes an additional endpoint (defaulting to `/graphql/stream`) used for [graphql-sse](https://github.com/enisdenjo/graphql-sse) clients.
+There are two different modes in [GraphQL over Server-Sent Events Protocol](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md). You can see the differences in the [protocol specification](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md).
+
+### Distinct Connection Mode
+
+GraphQL Yoga supports [GraphQL over Server-Sent Events Protocol](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md) only in distinct connection mode with an `sse.graphqlSSEDistinctConnections` flag.
+
+```ts filename="yoga.ts"
+import { createServer } from 'node:http'
+import { createYoga } from 'graphql-yoga'
+
+const yogaApp = createYoga({
+  // ...
+  sse: {
+    graphqlSSEDistinctConnections: true
+  }
+})
+```
+
+<Callout>
+  When you enable this, simple SSE recipes won't work because it changes the
+  data events format. You should use refer to [`graphql-sse` client
+  recipes](https://github.com/enisdenjo/graphql-sse#recipes).
+</Callout>
+
+### Single Connection Mode
+
+In case you want the subscriptions to be transported following the [GraphQL over Server-Sent Events Protocol](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md) also in the single connection mode, you simply use the `@graphql-yoga/plugin-graphql-sse` plugin for GraphQL Yoga that exposes an additional endpoint (defaulting to `/graphql/stream`) used for [graphql-sse](https://github.com/enisdenjo/graphql-sse) clients.
```

Comment:
```suggestion
In case you want the subscriptions to be transported following the [GraphQL over Server-Sent Events Protocol](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md#single-connection-mode) also in the single connection mode, you simply use the `@graphql-yoga/plugin-graphql-sse` plugin for GraphQL Yoga that exposes an additional endpoint (defaulting to `/graphql/stream`) used for [graphql-sse](https://github.com/enisdenjo/graphql-sse) clients.
```